spotify-app/demiv2/script/spoty-api-1.0.0.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Spotify API client.
+ *
+ * @version 1.0.0
+ */
+var spotifyApi;
```

Comment:
USE let instead of var here ,)

Reply:
Oops! sorry, it will be changed  with the new version with all the @VGamezz19 reviews!!